modules/service/src/main/scala/lucuma/odb/service/GuideEnvironmentService.scala

```diff
@@ -437,22 +440,22 @@ object GuideEnvironmentService {
               )
               .toRight(s"Invalid elevation range in observation $id.")
 
-          val fpu: Either[String, Either[GmosNorthFpu, GmosSouthFpu]] =
+          val fpu: Option[Either[GmosNorthFpu, GmosSouthFpu]] =
             (nFpu, sFpu) match {
-              case (Some(north), None) => north.asLeft.asRight
-              case (None, Some(south)) => south.asRight.asRight
-              case _                   => s"No configuration for observation $id.".asLeft
+              case (Some(north), None) => north.asLeft.some
+              case (None, Some(south)) => south.asRight.some
+              case _                   => none
             }
 
           val explicitBase: Option[Coordinates] =
             (ra, dec).mapN(Coordinates(_, _))
 
-          (elevRange, fpu).mapN((elev, f) =>
+          elevRange.map(elev =>
```

Comment:
Was the problem that they are `Either`s but switching to `Validated` and then back would work or that the message just doesn't make it up to where it needs to go?  If the later, do we fail to see the "Invalid elevation range in observation" message when it happens too?

Reply:
Yes, we would fail to see the `invalid elevation range in observation` message as well, but this should never happen, right? Observations are required to have contraints which include an elevation range, IIRC.